### DISCUSSION TITLE
Normally ON/OFF LED indicator

### DISCUSSION THREAD

**Discussion by colinku** [Pirate-MIDI/Pirate-MIDI-Features-Bug-Tracking#14] [Feature Requests & Ideas]
It would be sweet to have a patch level option to have a switch's LED on or off upon loading of a new bank under
Menu-->Switches-->Select Switch--> Config-->If Toggle or Momentary---> Column 4 (after colour) = LED State --> Normally On/Normally Off

**Comment by samspencer5991** [comment]
Great idea!
